library/src/CMakeLists.txt

```diff
@@ -180,6 +184,7 @@ if(HIP_PLATFORM STREQUAL amd)
     rocm_export_targets(
         TARGETS roc::hipblas
 	DEPENDS PACKAGE hip
+	STATIC_DEPENDS ${static_depends}
 	NAMESPACE roc::
```

Comment:
I guess this is empty list for CUDA backend?   Should probably add cuda static_depends as they already provide static libs so we can test already.  These changes aren't tested in CI so can you post test results @evetsso ?  Thanks.

Reply:
CUDA doesn't provide CMake infrastructure for its libraries.  I don't think there's much we can do on our end, since there is nothing we could add to either DEPENDS or STATIC_DEPENDS to make anything work better.

As for CI, I've run the hipBLAS regression tests manually with all-static-math-libs and (at least excluding the stress tests) they all pass.  Hopefully we can start to get static libs going on math-ci soon to help make sure this continues to work.